### PR TITLE
Print kubectl client version

### PR DESCRIPTION
**Pull request by dgageot**
Signed-off-by: David Gageot <david@gageot.net>

### CODE REVIEW

File: pkg/skaffold/deploy/kubectl.go [hunk 1]
```diff
@@ -64,6 +64,8 @@ func (k *KubectlDeployer) Labels() map[string]string {
 // Deploy templates the provided manifests with a simple `find and replace` and
 // runs `kubectl apply` on those manifests
 func (k *KubectlDeployer) Deploy(ctx context.Context, out io.Writer, builds []build.Artifact) ([]Artifact, error) {
+	logrus.Debugln("kubectl client version:", k.kubectl.Version())
```
Comment (nkubala) [inline pkg/skaffold/deploy/kubectl.go]: nit: I would even consider making this `Info` level log

Comment (dgageot) [reply pkg/skaffold/deploy/kubectl.go]: Good point

**Comment by dgageot** [comment]
@nkubala It should be all good now.

File: pkg/skaffold/deploy/kubectl/version.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		version := Version{
+			Client: ClientVersion{
+				Major: unknown,
+				Minor: unknown,
+			},
+		}
+
+		buf, err := c.getVersion(context.Background())
+		if err != nil {
+			logrus.Errorln("unable to get kubectl client version", err)
```
Comment (dlorenc) [inline pkg/skaffold/deploy/kubectl/version.go]: Should this be a warning instead of an error since it's not really fatal?

**Pull request merged**